src/test/app/LoanBroker_test.cpp

```diff
@@ -1015,6 +1015,13 @@ class LoanBroker_test : public beast::unit_test::suite
                 sig(sfCounterpartySignature, alice),
                 fee(env.current()->fees().base * 2));
 
+            // preflight: temINVALID (empty broker id)
+            {
+                auto jv = del(alice, brokerKeylet.key);
+                jv[sfLoanBrokerID] = "";
+                env(jv, ter(temINVALID));
+            }
```

Comment:
For completeness, could you add another test with an explicit zero?

```suggestion
            // preflight: temINVALID (empty broker id)
            {
                auto jv = del(alice, brokerKeylet.key);
                jv[sfLoanBrokerID] = "";
                env(jv, ter(temINVALID));
            }
            // preflight: temINVALID (zero broker id)
            {
                // needs a flag to distinguish the parsed STTx from the prior test
                auto jv = del(alice, uint256{}, 1);
                BEAST_EXPECT(
                    jv[sfLoanBrokerID] ==
                    "0000000000000000000000000000000000000000000000000000000000"
                    "000000");
                env(jv, ter(temINVALID));
            }
```